tf-modules/report-granules/index.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const get = require('lodash.get');
+const has = require('lodash.has');
+
+const {
+  isSnsEvent,
+  getSnsEventMessageObject
+} = require('@cumulus/common/sns-event');
+const Granule = require('@cumulus/api/models/granules');
+
+/**
+ * Check if message contains granules.
+ * Checks locations expected by sns2elasticsearch
+ *
+ * @param {Object} message - SNS Cumulus message object
+ * @returns {boolean} whether the message contains granules
+ */
+function containsGranules(message) {
+  return (has(message, 'payload.granules') || has(message, 'meta.input_granules'));
+}
+
+/**
+ * Process Cumulus message object and create granule database records.
+ *
+ * @param {*} message - SNS Cumulus message object
```

Comment:
Why the `*`? Because `getSnsEventMessageObject` in theory could return a non object?

Reply:
Missed when it auto-generated the doc.